portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,91 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#pf-image {
```

Comment:
I don't think this is used, but maybe it's for projects.html that I don't see.

Reply:
I believe this is actually used to style the image on index.html.